.gitignore

```diff
@@ -5,3 +5,1211 @@ npm/fbx2gltf/node_modules/
 npm/tests/node_modules/
 npm/tests/test/*.js
 npm/tests/test/*.js.map
+bin/linux/cmake_install.cmake
```

Comment:
was this auto-updated? I think it can just be bin/**!

Reply:
Yes this was auto updated! Good point I'll change that